packages/location-management/src/components/LocationForm/index.tsx

```diff
@@ -79,7 +79,7 @@ const defaultProps = {
   onCancel: () => void 0,
   service: OpenSRPService,
   username: '',
-  openSRPBaseURL: baseURL,
+  opensrpBaseURL: baseURL,
```

Comment:
I think this prop value needs to be changed on all other pages that consume this form, e.g. CreateServicePoint, EditServicePoint, NewLocationUnit, EditLocationUnit
on the same, wondering what standard we should use for such vars that contain abbreviation?
I vote for pure camelCase i.e `openSrpBaseUrl`.

Reply:
> because we have now switched to using pkg-config

should we implement that change in this pr, cause otherwise the creation pages will be broken for some instances that require a different baseURL

Reply:
If the creation pages are the only ones using `openSRPBaseURL` then i would suggest those be switched to `opensrpBaseURL` because all the other modules are using the latter

Reply:
And we should probably do that in a separate pr this one seems to be bloated enough

Reply:
that will mean we will have a faulty build in master, IMO that is something
 we should avoid.

Reply:
It wont fail because [here](https://github.com/OpenSRP/web/blob/master/app/src/App/utils.tsx#L40) is where the baseURL prop is defined and it seems the same is being used [here](https://github.com/OpenSRP/web/blob/master/packages/inventory/src/containers/CreateServicePoint/index.tsx#L24-L27) and also [here](https://github.com/OpenSRP/web/blob/master/packages/inventory/src/containers/EditServicePoint/index.tsx#L24-L27)

Reply:
addressed via https://github.com/OpenSRP/web/commit/0aa365e0fab551382091928f3c270e20c7e1e463